data/abilities.js

```diff
@@ -840,11 +840,17 @@ exports.BattleAbilities = {
 	},
 	"emergencyexit": {
 		shortDesc: "This Pokemon switches out when it reaches 1/2 or less of its maximum HP.",
-		// TODO
 		id: "emergencyexit",
 		name: "Emergency Exit",
 		rating: 3,
 		num: 194,
+		onHit: function (target) {
+			if (!this.canSwitch(target.side) || target.forceSwitchFlag) return;
+			if (target.hp <= target.maxhp/2) {
+				target.switchFlag = true;
```

Comment:
Oh thanks for the heads up on that, Marty had pointed this out a few minutes ago on main, but didnt elaborate on a possible fix